priv/local/client.ex

```diff
@@ -1,5 +1,6 @@
 # local client to be used in tests and also
 # dev env
 defmodule SupabasePotion.Client do
```

Comment:
At what point are we going to convert these to `SupabaseEx`?

Reply:
ditto on the `:supabase_potion` hex name, seems out of line with the repo name.

Reply:
sent you a slack message, but to summarize: i need explicit permission to retire the `supabase_potion` package form hex (7.6k downloads) with "renamed" reason and publish a new one `supabase` or `supabase_ex`. the first one is already taken by @treebee 

but about this comment itself, i think the module name could be only `Supabase` as already is, this file is only for testing purposes to avoid needing to create a new client instance every time